crates/red_knot_python_semantic/resources/mdtest/annotations/callable.md

```diff
@@ -63,7 +63,7 @@ from typing import Callable
 
 # error: [invalid-type-form] "Special form `typing.Callable` expected exactly two arguments (parameter types and return type)"
 def _(c: Callable[[int, str]]):
-    reveal_type(c)  # revealed: (int, str, /) -> Unknown
+    reveal_type(c)  # revealed: (...) -> Unknown
 ```
```

Comment:
`(int, str, /) -> Unknown` is defensible here, but ultimately `Callable[[int, str]]` is invalid as an annotation, and I would prefer us to have a consistent fallback inference for all invalid `Callable` annotations.